near/src/main/java/com/adroitandroid/near/connect/TcpServerService.java

```diff
@@ -1,15 +1,17 @@
 package com.adroitandroid.near.connect;
 
+import android.annotation.SuppressLint;
 import android.app.Service;
 import android.content.Context;
 import android.content.Intent;
 import android.os.Binder;
+import android.os.Build;
 import android.os.Handler;
 import android.os.HandlerThread;
 import android.os.IBinder;
 import android.os.Looper;
 import android.os.PowerManager;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
```

Comment:
at other places, we can use the NonNull sibling of this instead of from the jetbrains annotations